### ISSUE TITLE
Demo Fails - "No enum constant knox.spring.data.neo4j.domain.Edge.Orientation.inline"

### ISSUE THREAD

**Issue by Zackhardtoname** [CIDARLAB/knox#19]
Windows 10
Steps to reproduce

1. Download & Install Neo4j Browser
2. Follow the same steps in README - Quickstart (except running KnoxApplication.java from IntelliJ)
3. Run step 10 in Quickstart and it fails on joining test 1 with test 2, outputting the following errors in the console

`2019-07-05 13:59:50.619 ERROR 21664 --- [nio-8080-exec-6] o.a.c.c.C.[.[.[/].[dispatcherServlet]    : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is org.springframework.dao.InvalidDataAccessApiUsageException: Error mapping GraphModel to instance of knox.spring.data.neo4j.domain.DesignSpace; nested exception is org.neo4j.ogm.exception.MappingException: Error mapping GraphModel to instance of knox.spring.data.neo4j.domain.DesignSpace] with root cause

java.lang.IllegalArgumentException: No enum constant knox.spring.data.neo4j.domain.Edge.Orientation.inline`

On the webpage: 
![image](https://user-images.githubusercontent.com/25935197/60747863-69a5fb00-9f56-11e9-938f-b907a08e6948.png)

**Comment by dany-fu** [comment]
Is this bug only in the demo files? I recall you had another example where this occurred?

**Comment by Zackhardtoname** [comment]
Yes. They are all in the demo files. Since all the "orientation" property in neo4j had been in lower case, none of the operators would work. So it was really just one bug.
